raiden/utils/signing.py

```diff
@@ -0,0 +1,94 @@
+from coincurve import PrivateKey, PublicKey
```

Comment:
It may also be interesting to replace our private/public key conversion functions below, which uses coincurve, by `eth-keys` functions and methods, which is already pulled.

Reply:
perhaps but not in the scope of this PR

Reply:
@LefterisJP @andrevmatos eth-keys itself still uses coincurve, correct?

Reply:
It seems it has different [backends](https://github.com/ethereum/eth-keys/tree/master/eth_keys/backends)